package.json

```diff
@@ -37,5 +11,27 @@
   "bugs": {
     "url": "https://github.com/taskworld/rereselect/issues"
   },
-  "homepage": "https://github.com/taskworld/rereselect#readme"
+  "homepage": "https://github.com/taskworld/rereselect#readme",
+  "packageManager": "pnpm@9.4.0",
+  "main": "lib/index.js",
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "build": "rm -rf lib && tsc",
+    "test": "tsc --noEmit && vitest src/index.test.ts",
+    "bench": "vitest --run bench --mode production"
+  },
+  "devDependencies": {
+    "@types/lodash": "^4.17.6",
+    "lefthook": "^1.6.18",
+    "lodash": "^4.17.21",
+    "reselect3": "npm:reselect@^3.0.1",
+    "reselect5": "npm:reselect@^5.0.0",
+    "typescript": "^5.5.3",
+    "vitest": "^1.6.0"
+  },
+  "resolutions": {
+    "jsdom": "16.5.0"
```

Comment:
Somehow `vitest` has `jsdom` v11 as its dependency which `jsdom` has many other vul.
Therefore I have to override `jsdom` here.

Note that `vitest` and `jsdom` are NOT part of the production code.